Sources/Files.swift

```diff
@@ -298,6 +298,24 @@ public class FileSystem {
         return try! Folder(path: ProcessInfo.processInfo.homeFolderPath, using: fileManager)
     }
     
+    /// A reference to the document folder used by this file system
+    public var documentFolder: Folder? {
+        guard let url = try? fileManager.url(for: .documentDirectory, in: .userDomainMask, appropriateFor: nil, create: false) else { return nil }
+        
+        do {
+            return try Folder(path: url.path, using: fileManager)
```

Comment:
Sounds fair, since rest of the library heavily relies on `throws` instead of `nil` checks.